src/main/java/org/folio/rest/impl/PurchaseOrderLineHelper.java

```diff
@@ -156,7 +156,7 @@ private CompositePurchaseOrder validateOrderState(CompositePurchaseOrder po) {
           if (!config.isEmpty() && !config.getString(CREATE_INVENTORY).isEmpty()) {
```

Comment:
As far as I understand this issue happened because when `config.isEmpty()` is `true`, the logic went to `else` section and brand new completed `CompletableFuture` was created...
However when `config` has some values, the second condition 
```java
config.getString(CREATE_INVENTORY).isEmpty()
```
fails with NPE because `config.getString(CREATE_INVENTORY)` returns `null` so the `exceptionally` block completes `future` defined above. Probably better to replace second (or both) condition by
```java
StringUtils.isNotEmpty(config.getString(CREATE_INVENTORY))
```

Reply:
condition fixed